code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -198,7 +199,8 @@
 		new V(podman)
 	podman.hardset_dna(null,null,null,podman.real_name,blood_type, new /datum/species/pod,features)//Discard SE's and UI's, podman cloning is inaccurate, and always make them a podman
 	podman.set_cloned_appearance()
+	var/datum/species/species = podman.dna.species
+	species.exotic_blood = max(reagents_add)
```

Comment:
```suggestion
	podman.dna.species.exotic_blood = max(reagents_add) || /datum/reagent/water
```
Here's a suggestion for if you want their exotic blood to default to water
if max(reagents_add) returns null, either due to reagents_add being `null` itself or empty, it will default to `/datum/reagent/water`.